.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
       - name: Run chart-testing (lint)
-        uses: helm/chart-testing-action@master
+        uses: helm/chart-testing-action@main
         with:
           command: lint
           config: .github/ct-config.yml
```

Comment:
This is necessary, but not sufficient:

https://github.com/GarnerCorp/loki/actions/runs/1813862837





https://garnercorp.slack.com/archives/D32BTFS10/p1644343032265809
> Unexpected input(s) 'command', 'config', valid inputs are ['version']

https://github.com/GarnerCorp/loki/runs/5113756264?check_suite_focus=true#step:3:31

https://github.com/helm/chart-testing-action/blob/main/README.md#example-workflow